Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-slim
+
+WORKDIR /code
+
+COPY requirements.txt .
+
+RUN pip install -r  requirements.txt
+
+COPY . .
```

Comment:
why did you use COPY twice ? 
this is always used for caching any way

Reply:
I modified the dockerfile

Reply:
your previous Dockerfile was not wrong

Reply:
I know that but I don't want to use copy more than once